src/main/java/net/greatstart/dto/DtoUserProfile.java

```diff
@@ -0,0 +1,15 @@
+package net.greatstart.dto;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+
+@Data
+public class DtoUserProfile {
+    private Long id;
+    @NotNull
+    private String name;
+    private String address;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "address" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)